src/components/Form/Widgets/SelectWidget.tsx

```diff
@@ -18,7 +18,9 @@ export const SelectWidget = ({
 	multiple,
 	options,
 	required,
+	uiSchema,
 }: WidgetProps) => {
+	const { inputProps, ...otherUiSchema } = uiSchema ?? {};
```

Comment:
Perhaps we should pass `InputProps` instead and do `<TextField {...params} {...InputProps} />` for `renderInput` instead. I think there's some props we may want to change outside of the inner `inputProps` 🤔 Not certain

Reply:
@myarmolinsky not getting the comment sorry, could you please expand on what you think the solution could be ?

just bear in mind that:
- `params` also contains some inputProps that we don't want override
- inputProps and InputProps are different props https://mui.com/material-ui/api/text-field/#TextField-prop-inputProps